source/pip/tests/test_qasm.py

```diff
@@ -543,24 +543,6 @@ def test_callables_with_unsupported_types_raise_errors_on_call() -> None:
         code.Unsupported()
 
 
-def test_callables_with_unsupported_udt_types_raise_errors_on_call() -> None:
```

Comment:
Now that these don't trigger unsupported errors, should we have positive tests for these? Specifically, something that takes and/or returns the OpenQASM `complex` works as expected.

Reply:
Agreed, we should have positive tests for this now.